src/chat.rs

```diff
@@ -1586,6 +1597,56 @@ pub fn set_gossiped_timestamp(context: &Context, chat_id: u32, timestamp: i64) {
     }
 }
 
+pub fn shall_attach_selfavatar(context: &Context, chat_id: u32) -> Result<bool, Error> {
+    // versions before 12/2019 already allowed to set selfavatar, however, it was never sent to others.
+    // to avoid sending out previously set selfavatars unexpectedly we added this additional check.
+    // it can be removed after some time.
+    if !context
+        .sql
+        .get_raw_config_bool(context, "attach_selfavatar")
+    {
+        return Ok(false);
+    }
+
+    let resend_every_days = 14;
+    let timestamp_some_days_ago = time() - resend_every_days * 24 * 60 * 60;
```

Comment:
can we make these magic numbers constants please?

Reply:
yip, i do